docs/SimulationTutorial.md

```diff
@@ -53,7 +56,7 @@ Assuming already completed [Autoware setup](https://github.com/tier4/AutowareArc
 1. Download sample map from [here](https://drive.google.com/open?id=197kgRfSomZzaSbRrjWTx614le2qN-oxx) and extract the zip file.
 2. Launch Autoware with Planning Simulator
 
-```
+```sh
 source devel/setup.bash
```

Comment:
Should it be changed to `source install/setup.bash`?

Reply:
ah, yes, exactly!